### PR TITLE
Devrel

### PR DESCRIPTION
**Pull request by eric-giannini**
### PR Details
Updated the `telephony-api.md`
It lacked description of credential handling, incorrectly added `tracker_response`
**Link to original source**: https://github.com/symblai/symbl-docs/blob/develop/python-sdk/telephony-api.md

Closes #

### CODE REVIEW

File: python-sdk/telephony-api.md [hunk 1]
```diff
@@ -21,20 +24,39 @@ The Python SDK provides the following capabilities:
...
 
 
+## Credential Handling 
+
+Symbl.ai's Python SDK simplifies the credential handling by allowing you either to add your credentials directly to the connection method's calls or else through a separate file saved to your execution directory. 
+
+To add your credentials directly to the connection method's calls, add the following line: 
+
+```python
+      credentials={app_id: <app_id>, app_secret: <app_secret>}, #Optional, Don't add this parameter if you have symbl.conf file in your home directory or working directory
```
Comment (rishabh-chaturvedi) [inline python-sdk/telephony-api.md:34]: We can remove the comment from here.

File: python-sdk/telephony-api.md [hunk 2]
```diff
@@ -21,20 +24,39 @@ The Python SDK provides the following capabilities:
...
 
 The code snippet below allows you to start a Telephony connection with Symbl via PSTN protocol: 
 
 ```py
 import symbl
 
-phoneNumber = "" #Zoom phone number. Can be found in invitation 
-meetingId = "" # Your zoom meetingId
-password = "" # Your zoom meeting passcode
+phoneNumber = "" #A PSTN number or a Zoom number
```
Comment (rishabh-chaturvedi) [inline python-sdk/telephony-api.md:52]: Zoom is a good example for telephony API, however it's not limited to Zoom only. We can write "A valid US phone number" here. WDYT?

**Review by rishabh-chaturvedi — CHANGES_REQUESTED** [review]

File: python-sdk/telephony-api.md (hunk 2 above)
Comment (eric-giannini) [reply python-sdk/telephony-api.md:52]: 👍🏽

File: python-sdk/telephony-api.md (hunk 1 above)
Comment (eric-giannini) [reply python-sdk/telephony-api.md:34]: 👍🏽

**Review by pema-s — APPROVED** [review]

**Pull request merged**